README.md

```diff
@@ -4,13 +4,26 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1. The smaller order variables are ignored.
+  2. Coefficents are ignored.
+  3. Asymptotic analysis considers worse time which can be misleading.
```

Comment:
Worse time? Do you mean worst-case analysis? We can also do best and average case analysis.